Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -5948,7 +5948,7 @@ void WebPageProxy::decidePolicyForResponseShared(Ref<WebProcessProxy>&& process,
         ), webPageID);
         });
 #if USE(QUICK_LOOK)
-        if (process->captivePortalMode() == WebProcessProxy::CaptivePortalMode::Enabled && (MIMETypeRegistry::isPDFOrPostScriptMIMEType(navigationResponse->response().mimeType()) || PreviewConverter::supportsMIMEType(navigationResponse->response().mimeType())))
+        if (process->captivePortalMode() == WebProcessProxy::CaptivePortalMode::Enabled && (MIMETypeRegistry::isPDFOrPostScriptMIMEType(navigationResponse->response().mimeType()) || MIMETypeRegistry::isSupportedModelMIMEType(navigationResponse->response().mimeType()) || PreviewConverter::supportsMIMEType(navigationResponse->response().mimeType())))
```

Comment:
Not new to this patch but I think this is slightly wrong. What if the client app responded with PolicyAction::Ignore ? Then we are going to download the file instead of ignoring it like the client asked...

I think we should only override to `PolicyAction ::Download` if `policyAction` is `PolicyAction::Use`.